### ISSUE TITLE
False positive on unirest-java-1.4.9

### ISSUE THREAD

**Issue by sichapman** [dependency-check/DependencyCheck#1092]
False positive on library unirest-java-1.4.9.jar - reported as cpe:/a:rest-client_project:rest-client:1.4.9

Therefore CVE-2015-1820 and CVE-2015-3448 get reported

```xml
<dependency>
    <groupId>com.mashape.unirest</groupId>
    <artifactId>unirest-java</artifactId>
    <version>1.4.9</version>
</dependency>
```

**Comment by jeremylong** [comment]
Thanks for the FP report

**Comment by jeremylong** [comment]
This was resolved in 3.1.2.
